_episodes/41-code-review.md

```diff
@@ -278,127 +269,140 @@ levels of documentation?
+
+#### Adding a review comment
+
+1. To be able to add your review to a pull request created on your course mate's repository, 
+they will need to add you as a collaborator on their repository first. They can do that from 
+the `Settings` tab of the repository, then `Collaborators and teams` tab on the left, 
+then clicking the `Add people` button. 
+Once they find you by your GitHub name - you will receive an invitation via email to join the 
+repository as a collaborator. 
+You will have to do the same for the collaborator doing the review on your repository.
+![Adding a collaborator in GitHub](../fig/github-add-collaborator.png){: .image-with-shadow width="900px"}
+2. Open up the pull request from the GitHub's `Pull Requests` tab on the home page 
+of your course mate's software repository, then head to the `Files changed` tab.
+![The files changed tab of a pull request](../fig/github-pull-request-files-changed.png){: .image-with-shadow width="900px"}
+3. When you find a line that you want to add a comment to, click on the blue
+plus (+) button next to the line. This will bring up a "Write" box to add your comment.
+![Adding a review comment to a pull request](../fig/github-pull-request-add-comment.png){: .image-with-shadow width="800px"}
 You can also add comments referring to multiple lines by clicking the plus and
 dragging down over the relevant lines.
-
-If you want to make a concrete suggestion, such as renaming a variable, you
-can click the `Add a suggestion button` (which looks like a document with a plus and a minus in it).
+If you want to make a concrete suggestion or a change to the code directly, 
+such as renaming a variable, you can click the `Add a suggestion` button 
+(which looks like a document with a plus and a minus in it).
 This will populate the comment with the existing code, and you can edit it to be
 what you think the code should be.
-![Adding a suggestion to a pull request](../fig/github-pull-request-add-suggestion.png){: .image-with-shadow }
-GitHub will then provide a button for the author to apply your change directly.
-
-Write your comment in the box, and then click `Start review`.
+![Adding a suggestion to a pull request](../fig/github-pull-request-add-suggestion.png){: .image-with-shadow width="800px"} 
+GitHub will then provide a button for the code author to apply your change directly.
+4. Write your comment in the box, and then click `Start review`.
 This will save your comment, but not publish it yet.
-
-You can use `Add single comment` to immediately post a comment.
+You can use `Add single comment` button to immediately post a comment.
 However, it is best to batch the comments into a single review, so that the author
 knows when you have finished adding comments
 (and avoid spamming their email with notifications).
-
-Continue adding comments in this way, using the `Add review comment` button
+5. Continue adding comments in this way, using the `Add review comment` button
```

Comment:
Looking at the original PR, this is supposed to be done in a group in the following exercise, so probably should be removed.

Reply:
Think it is OK to keep it as is as reviewers may have more comments to add here, before we move to the next exercise and they even more comments around tests @steve-crouch

Reply:
Improved via f51ee2e2639cb1540aec389f6ab060389c49c2a2